addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -63,9 +63,6 @@ def get_operations(self, bom_id=False, qty=0, level=0):
         }
         return self.env.ref('mrp.report_mrp_operation_line').render({'data': values})
 
-    def _get_bom_reference(self, bom):
-        return bom.display_name
```

Comment:
Sad. Each time a hook is deleted, each time a cat miserably dies.

Reply:
You can do as many hook as you want with _get_bom and _get_bom_line. This was a useless hook